src/js/blocks/hero/block.js

```diff
@@ -291,21 +284,22 @@ registerBlockType( 'uri-cl/hero', {
 						<PanelRow>
 							<BaseControl
 								label={ __( 'Format' ) }
+								id="hero-format"
 							>
 								<ButtonGroup aria-label={ __( 'Hero Format' ) }>
 									{ [ 'default', 'fullwidth', 'super' ].map( ( value ) => {
 										const capitalizedValue = value.charAt( 0 ).toUpperCase() + value.slice( 1 );
 										const key = ( 'default' === value ) ? '' : value;
-										const format = ( undefined == attributes.format ) ? '' : attributes.format;
-										const isSelected = ( key === format );
+										const format = ( undefined === attributes.format ) ? '' : attributes.format;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of undefined.](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4811013)